build/build.xml

```diff
@@ -386,6 +394,7 @@
 		<build-addon name="cmss" />
 		<build-addon name="codedx"/>
 		<build-addon name="communityScripts"/>
+		<build-addon name="cspscanner"/>
```

Comment:
I did, sorry, I should have said - the lack of this line means that the cspscanner doesnt get included in the weekly build, even though it sound be (see the deploy-weekly target)